docs/commands/locale.md

```diff
@@ -0,0 +1,72 @@
+# `/locale` command
+
+Check or set Bastion's locale for this channel or server. _This command is in development._
+
+This setting is meant to apply to all text displayed by the new instance of Bastion, but
+translations may not be complete, in which case it will fall back to English. Features
+still handled by the old instance will not be affected.
+
+## Definitions
+
+**Locale**: a parameter affecting _localization_, such as what language and dialect are used,
```

Comment:
are we using -ization or -isation? I'm used to -isation and I thought we were using en-AU, but en-AU isn't strictly en-GB and I could believe -ization is technically correct.

Reply:
Oops it should be _isation_ for both, it's Canadian follows the American spelling for this one.